content/join: Revise the joining instructions (#8)
coderefinery/2022-03-22-workshop: content/join.md

  we can (up to the number of exercise leaders who volunteer)
- If a **team** registers with their own exercise leader, you get
  priority and we will try to accept the whole team.  Everyone on the
  team should register separately and set the same `Team name` in the
  registration.

Attending via Zoom will give you everything the livestream does, with
the possibly of centrally organized teams and a bit more help.

### FAQ: how should I register?

- **I am alone and want to work with others, how should I register?**
  - Register for Zoom (if we are full you get the livestream anyway)
- **Why teams?**
  - People learn best in small groups, even if the primary teaching
    sessions can reach many people and be effective.  Our strategy
    gives us the best of both worlds.
- **I am making my own team, how should I register?**
  - **Want more personal support:** `Zoom` you will have more
    support from our expert helpers, which may be good for less
    confident teams.
  - **Want to manage everything yourself** (for example make your own
    Zoom or in-person room): `Livestream only`.  The coordinator can
    still join our CodeRefinery chat and interact with us.

>   - The coordinator could register on behalf of your team and forward

Reviewer: isn't this contradictory with the point below, which encourages everyone to register so that CR can better measure the environment?

Author: Is it better now?  I tried to do some sort of progressive disclosure, make it simple then make it more complex.  But yeah, it's not good if it's inconsistent.  Is this good?

Reviewer: yes I think now it's good

>     information to the others.

Author: ```suggestion
    information to the others (but it's nice if everyone registers, see below).
```

- **My complete team will work together in a physical room, is there
  any point to joining your Zoom?**
  - Expert helpers will drop by the Zoom breakout room and ask if
    things are going OK.  If you won't want to screenshare with them
    and see their demos, there is probably no point.  (Note that Zoom
    is still useful in the physical room for screensharing with each
    other).

> - **I want to make my a team on my own, do I need to register?  Does

Reviewer: ```suggestion
- **I want to make my team on my own, do I need to register?  Does
```

  everyone on the team need to register?**
  - Technically you don't have to, if you carefully check the website
    for news.
  - We prefer if you register so that we can report our impact better.
    Also, the team gets emails direct from us and the HackMD links to
    ask questions.
- **I'm an organization or group, I want to sponsor my community's teams!**

>   - Great!  You can do this with our without communicating with us.

Reviewer: ```suggestion
  - Great!  You can do this with or without communicating with us.
```

    Feel free to just make your own registration form and manage it if

>     you want.  Or talk with us organize them in our central Zoom.

Author: ```suggestion
    you want.  Or talk with us organize them as part of our centrally organized Zoom.
```

>     you want.  Or talk with us organize them in our central Zoom.

Reviewer: is there a preposition missing here?
